index.html

```diff
@@ -17,32 +17,23 @@
           name: "Marcos Cáceres",
           company: "W3C",
           companyURL: "https://w3.org",
-          w3cid: 39125,
+          w3cid: "39125",
         },
       ],
       formerEditors: [
         {
           name: "Andrei Popescu",
           company: "Google Inc.",
           companyURL: "https://google.com/",
-          w3cid: "36565",
         },
       ],
       group: "das",
-      previousPublishDate: "2016-11-08",
-      previousMaturity: "REC",
+      previousPublishDate: "2021-06-07",
+      previousMaturity: "FPWD",
       github: "w3c/geolocation-api",
       caniuse: "geolocation",
       testSuiteURI: "https://wpt.live/geolocation-API/",
-      // implementationReportURI: "https://wpt.fyi/geolocation-API",
-      localBiblio: {
-        WGS84: {
```

Comment:
Looks like this biblio entry is still needed.

Reply:
Sorry, I should have added a note: https://github.com/tobie/specref/pull/671